sdk/storage/test/test_base.cpp

```diff
@@ -151,4 +163,12 @@ namespace Azure { namespace Storage { namespace Test {
 
   void DeleteFile(const std::string& filename) { std::remove(filename.data()); }
 
+  std::vector<uint8_t> RandomBuffer(std::size_t length)
+  {
+    std::vector<uint8_t> result(length);
+    uint8_t* dataPtr = const_cast<uint8_t*>(result.data());
```

Comment:
`const_cast` is not something we want

Reply:
It's in test code and I don't want to spend all day reimplementing test utilities.